Dockerfile.openssl

```diff
@@ -28,19 +28,6 @@ RUN mkdir QAT_Lib && \
     install -m 755 build/libusdm_drv_s.so /usr/lib/ && \
     install -m 755 build/adf_ctl /usr/bin/
 
-WORKDIR /
```

Comment:
the current ordering was made on purpose (it was the ipp-crypto/ipsec-mb changing frequently) but I guess things have settled and this change makes sense. 

Perhaps move this commit later in the series because you're going to modify some of these lines later still

Reply:
@mythi 
> Perhaps move this commit later in the series because you're going to modify some of these lines later still

Makes sense. OK, moved those lines into 0eca540, please take a look.

Thanks for the review!